activerecord/test/cases/relations_test.rb

```diff
@@ -1911,15 +1911,8 @@ def test_presence
   end
 
   test "using a custom table affects the wheres" do
-    table_alias = Post.arel_table.alias("omg_posts")
-
-    table_metadata = ActiveRecord::TableMetadata.new(Post, table_alias)
-    predicate_builder = ActiveRecord::PredicateBuilder.new(table_metadata)
-    relation = ActiveRecord::Relation.new(Post, table_alias, predicate_builder)
-    relation.where!(foo: "bar")
-
-    node = relation.arel.constraints.first.grep(Arel::Attributes::Attribute).first
-    assert_equal table_alias, node.relation
+    post = posts(:welcome)
+    assert_equal post, OmgPost.where(title: post.title).take
```

Comment:
This seems like it's just a cosmetic change. It also makes it much less clear from the code what's being tested.